<Type Name="DivideByZeroException" FullName="System.DivideByZeroException">
  <Metadata>
    <Meta Name="ms.openlocfilehash" Value="ab52232ecf30dfd61be3fc6b47f6349b67187a91" />
    <Meta Name="ms.sourcegitcommit" Value="d31dc2ede16f6f7bc64e90d9f897ff54c4e3869b" />
    <Meta Name="ms.translationtype" Value="HT" />
    <Meta Name="ms.contentlocale" Value="ja-JP" />
    <Meta Name="ms.lasthandoff" Value="04/03/2018" />
  </Metadata>
  <TypeSignature Language="C#" Value="public class DivideByZeroException : ArithmeticException" />
  <TypeSignature Language="ILAsm" Value=".class public auto ansi serializable beforefieldinit DivideByZeroException extends System.ArithmeticException" />
  <TypeSignature Language="DocId" Value="T:System.DivideByZeroException" />
  <TypeSignature Language="VB.NET" Value="Public Class DivideByZeroException&#xA;Inherits ArithmeticException" />
  <TypeSignature Language="C++ CLI" Value="public ref class DivideByZeroException : ArithmeticException" />
  <AssemblyInfo>
    <AssemblyName>System.Runtime</AssemblyName>
    <AssemblyVersion>4.0.0.0</AssemblyVersion>
    <AssemblyVersion>4.0.10.0</AssemblyVersion>
    <AssemblyVersion>4.0.20.0</AssemblyVersion>
    <AssemblyVersion>4.1.0.0</AssemblyVersion>
    <AssemblyVersion>4.2.0.0</AssemblyVersion>
    <AssemblyVersion>4.2.1.0</AssemblyVersion>
  </AssemblyInfo>
  <AssemblyInfo>
    <AssemblyName>mscorlib</AssemblyName>
    <AssemblyVersion>2.0.5.0</AssemblyVersion>
    <AssemblyVersion>4.0.0.0</AssemblyVersion>
  </AssemblyInfo>
  <AssemblyInfo>
    <AssemblyName>netstandard</AssemblyName>
    <AssemblyVersion>2.0.0.0</AssemblyVersion>
  </AssemblyInfo>
  <Base>
    <BaseTypeName>System.ArithmeticException</BaseTypeName>
  </Base>
  <Interfaces />
  <Attributes>
    <Attribute>
      <AttributeName>System.Runtime.InteropServices.ComVisible(true)</AttributeName>
    </Attribute>
  </Attributes>
  <Docs>
    <summary>
      <span data-ttu-id="3e321-101">整数または <see cref="T:System.Decimal" /> 値を 0 で除算しようとするとスローされる例外。</span>
      <span class="sxs-lookup">
        <span data-stu-id="3e321-101">The exception that is thrown when there is an attempt to divide an integral or <see cref="T:System.Decimal" /> value by zero.</span>
      </span>
    </summary>
    <remarks>
      <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="3e321-102">整数の除算を試みるまたは<xref:System.Decimal>ゼロをスローして番号、<xref:System.DivideByZeroException>例外。</span><span class="sxs-lookup"><span data-stu-id="3e321-102">Trying to divide an integer or <xref:System.Decimal> number by zero throws a <xref:System.DivideByZeroException> exception.</span></span> <span data-ttu-id="3e321-103">例外を防ぐためには、ことを確認する整数値で除算の分母または<xref:System.Decimal>値は 0 以外。</span><span class="sxs-lookup"><span data-stu-id="3e321-103">To prevent the exception, ensure that the denominator in a division operation with integer or <xref:System.Decimal> values is non-zero.</span></span>  
  
 <span data-ttu-id="3e321-104">例外をスローしない浮動小数点値を 0 で除算します。IEEE 754 演算の規則に従って結果正の無限大、負の無限大、または非数 (NaN) になります。</span><span class="sxs-lookup"><span data-stu-id="3e321-104">Dividing a floating-point value by zero doesn't throw an exception; it results in positive infinity, negative infinity, or not a number (NaN), according to the rules of IEEE 754 arithmetic.</span></span> <span data-ttu-id="3e321-105">次の例では、整数の除算ではなく、浮動小数点除算を使用するため、操作はスローされません、<xref:System.DivideByZeroException>例外。</span><span class="sxs-lookup"><span data-stu-id="3e321-105">Because the following example uses floating-point division rather than integer division, the operation does not throw a <xref:System.DivideByZeroException> exception.</span></span>  
  
 [!code-csharp[System.DivideByZeroException.Class#2](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.dividebyzeroexception.class/cs/exception2.cs#2)]
 [!code-vb[System.DivideByZeroException.Class#2](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.dividebyzeroexception.class/vb/exception2.vb#2)]  
  
 <span data-ttu-id="3e321-106">詳細については、<xref:System.Single> および <xref:System.Double> を参照してください。</span><span class="sxs-lookup"><span data-stu-id="3e321-106">For more information, see <xref:System.Single> and <xref:System.Double>.</span></span>  
  
 <span data-ttu-id="3e321-107">次の Microsoft intermediate language (MSIL) の命令をスロー <xref:System.DivideByZeroException>:</span><span class="sxs-lookup"><span data-stu-id="3e321-107">The following Microsoft intermediate language (MSIL) instructions throw <xref:System.DivideByZeroException>:</span></span>  
  
-   <span data-ttu-id="3e321-108">div</span><span class="sxs-lookup"><span data-stu-id="3e321-108">div</span></span>  
  
-   <span data-ttu-id="3e321-109">div.un</span><span class="sxs-lookup"><span data-stu-id="3e321-109">div.un</span></span>  
  
-   <span data-ttu-id="3e321-110">rem</span><span class="sxs-lookup"><span data-stu-id="3e321-110">rem</span></span>  
  
-   <span data-ttu-id="3e321-111">rem.un</span><span class="sxs-lookup"><span data-stu-id="3e321-111">rem.un</span></span>  
  
 <span data-ttu-id="3e321-112"><xref:System.DivideByZeroException> 値は 0x80020012 HRESULT COR_E_DIVIDEBYZERO を使用します。</span><span class="sxs-lookup"><span data-stu-id="3e321-112"><xref:System.DivideByZeroException> uses the HRESULT COR_E_DIVIDEBYZERO, which has the value 0x80020012.</span></span>  
  
 <span data-ttu-id="3e321-113">インスタンスの初期プロパティ値の一覧については<xref:System.DivideByZeroException>を参照してください、<xref:System.DivideByZeroException.%23ctor%2A>コンス トラクターです。</span><span class="sxs-lookup"><span data-stu-id="3e321-113">For a list of initial property values for an instance of <xref:System.DivideByZeroException>, see the <xref:System.DivideByZeroException.%23ctor%2A> constructors.</span></span>  
  
   
  
## Examples  
 <span data-ttu-id="3e321-114">次の例のハンドル、<xref:System.DivideByZeroException>整数除算の例外。</span><span class="sxs-lookup"><span data-stu-id="3e321-114">The following example handles a <xref:System.DivideByZeroException> exception in integer division.</span></span>  
  
 [!code-csharp[System.DivideByZeroException.Class#1](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.dividebyzeroexception.class/cs/exception1.cs#1)]
 [!code-vb[System.DivideByZeroException.Class#1](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.dividebyzeroexception.class/vb/exception1.vb#1)]  
  
 ]]></format>
    </remarks>
    <altmember cref="T:System.Exception" />
  </Docs>
  <Members>
    <MemberGroup MemberName=".ctor">
      <AssemblyInfo>
        <AssemblyName>System.Runtime</AssemblyName>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
        <AssemblyVersion>4.0.10.0</AssemblyVersion>
        <AssemblyVersion>4.0.20.0</AssemblyVersion>
        <AssemblyVersion>4.1.0.0</AssemblyVersion>
        <AssemblyVersion>4.2.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Docs>
        <summary>
          <span data-ttu-id="3e321-115">
            <see cref="T:System.DivideByZeroException" /> クラスの新しいインスタンスを初期化します。</span>
          <span class="sxs-lookup">
            <span data-stu-id="3e321-115">Initializes a new instance of the <see cref="T:System.DivideByZeroException" /> class.</span>
          </span>
        </summary>
      </Docs>
    </MemberGroup>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public DivideByZeroException ();" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor() cil managed" />
      <MemberSignature Language="DocId" Value="M:System.DivideByZeroException.#ctor" />
      <MemberSignature Language="VB.NET" Value="Public Sub New ()" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; DivideByZeroException();" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.Runtime</AssemblyName>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
        <AssemblyVersion>4.0.10.0</AssemblyVersion>
        <AssemblyVersion>4.0.20.0</AssemblyVersion>
        <AssemblyVersion>4.1.0.0</AssemblyVersion>
        <AssemblyVersion>4.2.0.0</AssemblyVersion>
        <AssemblyVersion>4.2.1.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>mscorlib</AssemblyName>
        <AssemblyVersion>2.0.5.0</AssemblyVersion>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>netstandard</AssemblyName>
        <AssemblyVersion>2.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters />
      <Docs>
        <summary>
          <span data-ttu-id="3e321-116">
            <see cref="T:System.DivideByZeroException" /> クラスの新しいインスタンスを初期化します。</span>
          <span class="sxs-lookup">
            <span data-stu-id="3e321-116">Initializes a new instance of the <see cref="T:System.DivideByZeroException" /> class.</span>
          </span>
        </summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="3e321-117">このコンス トラクター、<xref:System.Exception.Message%2A>エラーを説明するシステム提供のメッセージの新しいインスタンスのプロパティなど「しようと 0 で除算します」。</span><span class="sxs-lookup"><span data-stu-id="3e321-117">This constructor initializes the <xref:System.Exception.Message%2A> property of the new instance to a system-supplied message that describes the error, such as "Attempted to divide by zero."</span></span> <span data-ttu-id="3e321-118">このメッセージには、現在のシステムのカルチャが考慮されます。</span><span class="sxs-lookup"><span data-stu-id="3e321-118">This message takes into account the current system culture.</span></span>  
  
 <span data-ttu-id="3e321-119"><xref:System.DivideByZeroException> のインスタンスの初期プロパティ値を次の表に示します。</span><span class="sxs-lookup"><span data-stu-id="3e321-119">The following table shows the initial property values for an instance of <xref:System.DivideByZeroException>.</span></span>  
  
|<span data-ttu-id="3e321-120">プロパティ</span><span class="sxs-lookup"><span data-stu-id="3e321-120">Property</span></span>|<span data-ttu-id="3e321-121">[値]</span><span class="sxs-lookup"><span data-stu-id="3e321-121">Value</span></span>|  
|--------------|-----------|  
|<xref:System.Exception.InnerException%2A>|<span data-ttu-id="3e321-122">null 参照 (Visual Basic の場合は `Nothing`)。</span><span class="sxs-lookup"><span data-stu-id="3e321-122">A null reference (`Nothing` in Visual Basic).</span></span>|  
|<xref:System.Exception.Message%2A>|<span data-ttu-id="3e321-123">ローカライズされたエラー メッセージ文字列。</span><span class="sxs-lookup"><span data-stu-id="3e321-123">The localized error message string.</span></span>|  
  
 ]]></format>
        </remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public DivideByZeroException (string message);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(string message) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.DivideByZeroException.#ctor(System.String)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (message As String)" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; DivideByZeroException(System::String ^ message);" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.Runtime</AssemblyName>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
        <AssemblyVersion>4.0.10.0</AssemblyVersion>
        <AssemblyVersion>4.0.20.0</AssemblyVersion>
        <AssemblyVersion>4.1.0.0</AssemblyVersion>
        <AssemblyVersion>4.2.0.0</AssemblyVersion>
        <AssemblyVersion>4.2.1.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>mscorlib</AssemblyName>
        <AssemblyVersion>2.0.5.0</AssemblyVersion>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>netstandard</AssemblyName>
        <AssemblyVersion>2.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="message" Type="System.String" />
      </Parameters>
      <Docs>
        <param name="message">
          <span data-ttu-id="3e321-124">エラーを説明する <see cref="T:System.String" />。</span>
          <span class="sxs-lookup">
            <span data-stu-id="3e321-124">A <see cref="T:System.String" /> that describes the error.</span>
          </span>
        </param>
        <summary>
          <span data-ttu-id="3e321-125">指定したエラー メッセージを使用して、<see cref="T:System.DivideByZeroException" /> クラスの新しいインスタンスを初期化します。</span>
          <span class="sxs-lookup">
            <span data-stu-id="3e321-125">Initializes a new instance of the <see cref="T:System.DivideByZeroException" /> class with a specified error message.</span>
          </span>
        </summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="3e321-126">`message` の内容は、ユーザーが理解できる内容にします。</span><span class="sxs-lookup"><span data-stu-id="3e321-126">The content of `message` is intended to be understood by humans.</span></span> <span data-ttu-id="3e321-127">このコンストラクターの呼び出し元は、この文字列が現在のシステムのカルチャに合わせてローカライズ済みであることを確認しておく必要があります。</span><span class="sxs-lookup"><span data-stu-id="3e321-127">The caller of this constructor is required to ensure that this string has been localized for the current system culture.</span></span>  
  
 <span data-ttu-id="3e321-128"><xref:System.DivideByZeroException> のインスタンスの初期プロパティ値を次の表に示します。</span><span class="sxs-lookup"><span data-stu-id="3e321-128">The following table shows the initial property values for an instance of <xref:System.DivideByZeroException>.</span></span>  
  
|<span data-ttu-id="3e321-129">プロパティ</span><span class="sxs-lookup"><span data-stu-id="3e321-129">Property</span></span>|<span data-ttu-id="3e321-130">[値]</span><span class="sxs-lookup"><span data-stu-id="3e321-130">Value</span></span>|  
|--------------|-----------|  
|<xref:System.Exception.InnerException%2A>|<span data-ttu-id="3e321-131">null 参照 (Visual Basic の場合は `Nothing`)。</span><span class="sxs-lookup"><span data-stu-id="3e321-131">A null reference (`Nothing` in Visual Basic).</span></span>|  
|<xref:System.Exception.Message%2A>|<span data-ttu-id="3e321-132">エラー メッセージ文字列。</span><span class="sxs-lookup"><span data-stu-id="3e321-132">The error message string.</span></span>|  
  
 ]]></format>
        </remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="protected DivideByZeroException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);" />
      <MemberSignature Language="ILAsm" Value=".method familyhidebysig specialname rtspecialname instance void .ctor(class System.Runtime.Serialization.SerializationInfo info, valuetype System.Runtime.Serialization.StreamingContext context) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.DivideByZeroException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)" />
      <MemberSignature Language="VB.NET" Value="Protected Sub New (info As SerializationInfo, context As StreamingContext)" />
      <MemberSignature Language="C++ CLI" Value="protected:&#xA; DivideByZeroException(System::Runtime::Serialization::SerializationInfo ^ info, System::Runtime::Serialization::StreamingContext context);" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>mscorlib</AssemblyName>
        <AssemblyVersion>2.0.5.0</AssemblyVersion>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>netstandard</AssemblyName>
        <AssemblyVersion>2.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>System.Runtime</AssemblyName>
        <AssemblyVersion>4.2.0.0</AssemblyVersion>
        <AssemblyVersion>4.2.1.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="info" Type="System.Runtime.Serialization.SerializationInfo" />
        <Parameter Name="context" Type="System.Runtime.Serialization.StreamingContext" />
      </Parameters>
      <Docs>
        <param name="info">
          <span data-ttu-id="3e321-133">シリアル化されたオブジェクト データを保持するオブジェクト。</span>
          <span class="sxs-lookup">
            <span data-stu-id="3e321-133">The object that holds the serialized object data.</span>
          </span>
        </param>
        <param name="context">
          <span data-ttu-id="3e321-134">転送元または転送先に関するコンテキスト情報。</span>
          <span class="sxs-lookup">
            <span data-stu-id="3e321-134">The contextual information about the source or destination.</span>
          </span>
        </param>
        <summary>
          <span data-ttu-id="3e321-135">シリアル化したデータを使用して、<see cref="T:System.DivideByZeroException" /> クラスの新しいインスタンスを初期化します。</span>
          <span class="sxs-lookup">
            <span data-stu-id="3e321-135">Initializes a new instance of the <see cref="T:System.DivideByZeroException" /> class with serialized data.</span>
          </span>
        </summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="3e321-136">このコンストラクターは、逆シリアル化中に、ストリームで送信された例外オブジェクトを再構築するために呼び出されます。</span><span class="sxs-lookup"><span data-stu-id="3e321-136">This constructor is called during deserialization to reconstitute the exception object transmitted over a stream.</span></span>  
  
 ]]></format>
        </remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public DivideByZeroException (string message, Exception innerException);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(string message, class System.Exception innerException) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.DivideByZeroException.#ctor(System.String,System.Exception)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (message As String, innerException As Exception)" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; DivideByZeroException(System::String ^ message, Exception ^ innerException);" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.Runtime</AssemblyName>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
        <AssemblyVersion>4.0.10.0</AssemblyVersion>
        <AssemblyVersion>4.0.20.0</AssemblyVersion>
        <AssemblyVersion>4.1.0.0</AssemblyVersion>
        <AssemblyVersion>4.2.0.0</AssemblyVersion>
        <AssemblyVersion>4.2.1.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>mscorlib</AssemblyName>
        <AssemblyVersion>2.0.5.0</AssemblyVersion>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>netstandard</AssemblyName>
        <AssemblyVersion>2.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="message" Type="System.String" />
        <Parameter Name="innerException" Type="System.Exception" />
      </Parameters>
      <Docs>
        <param name="message">
          <span data-ttu-id="3e321-137">例外の原因を説明するエラー メッセージ。</span>
          <span class="sxs-lookup">
            <span data-stu-id="3e321-137">The error message that explains the reason for the exception.</span>
          </span>
        </param>
        <param name="innerException">
          <span data-ttu-id="3e321-138">現在の例外の原因となった例外。</span>
          <span class="sxs-lookup">
            <span data-stu-id="3e321-138">The exception that is the cause of the current exception.</span>
          </span>
          <span data-ttu-id="3e321-139">
            <c>innerException</c> パラメーターが <see langword="null" /> でない場合は、内部例外を処理する <see langword="catch" /> ブロックで現在の例外が発生します。</span>
          <span class="sxs-lookup">
            <span data-stu-id="3e321-139">If the <c>innerException</c> parameter is not <see langword="null" />, the current exception is raised in a <see langword="catch" /> block that handles the inner exception.</span>
          </span>
        </param>
        <summary>
          <span data-ttu-id="3e321-140">指定したエラー メッセージおよびこの例外の原因となった内部例外への参照を使用して、<see cref="T:System.DivideByZeroException" /> クラスの新しいインスタンスを初期化します。</span>
          <span class="sxs-lookup">
            <span data-stu-id="3e321-140">Initializes a new instance of the <see cref="T:System.DivideByZeroException" /> class with a specified error message and a reference to the inner exception that is the cause of this exception.</span>
          </span>
        </summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="3e321-141">前の例外の直接の結果としてスローされる例外については、<xref:System.Exception.InnerException%2A> プロパティに、前の例外への参照が格納されます。</span><span class="sxs-lookup"><span data-stu-id="3e321-141">An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the <xref:System.Exception.InnerException%2A> property.</span></span> <span data-ttu-id="3e321-142"><xref:System.Exception.InnerException%2A> プロパティは、コンストラクターに渡されたものと同じ値を返します。`null` プロパティによって内部例外値がコンストラクターに渡されなかった場合は、<xref:System.Exception.InnerException%2A> を返します。</span><span class="sxs-lookup"><span data-stu-id="3e321-142">The <xref:System.Exception.InnerException%2A> property returns the same value that is passed into the constructor, or `null` if the <xref:System.Exception.InnerException%2A> property does not supply the inner exception value to the constructor.</span></span>  
  
 <span data-ttu-id="3e321-143"><xref:System.DivideByZeroException> のインスタンスの初期プロパティ値を次の表に示します。</span><span class="sxs-lookup"><span data-stu-id="3e321-143">The following table shows the initial property values for an instance of <xref:System.DivideByZeroException>.</span></span>  
  
|<span data-ttu-id="3e321-144">プロパティの型</span><span class="sxs-lookup"><span data-stu-id="3e321-144">Property Type</span></span>|<span data-ttu-id="3e321-145">条件</span><span class="sxs-lookup"><span data-stu-id="3e321-145">Condition</span></span>|  
|-------------------|---------------|  
|<xref:System.Exception.InnerException%2A>|<span data-ttu-id="3e321-146">内部例外の参照。</span><span class="sxs-lookup"><span data-stu-id="3e321-146">The inner exception reference.</span></span>|  
|<xref:System.Exception.Message%2A>|<span data-ttu-id="3e321-147">エラー メッセージ文字列。</span><span class="sxs-lookup"><span data-stu-id="3e321-147">The error message string.</span></span>|  
  
 ]]></format>
        </remarks>
        <altmember cref="T:System.Exception" />
      </Docs>
    </Member>
  </Members>
</Type>